modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/scim/IDENTITY4776/catalina_server_config.toml

```diff
@@ -33,7 +32,7 @@ name = "wso2carbon.jks"
 password = "wso2carbon"
 
 [transport.https]
-properties.maxThreads = "1"
+maxThreads = "1"
 
 [transport.http]
-properties.maxThreads = "1"
\ No newline at end of file
+maxThreads = "1"
```

Comment:
Add new line here

Reply:
fix with https://github.com/wso2/product-is/pull/6803